README.md

```diff
@@ -540,11 +540,15 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 [Rust Embedded](https://rust-embedded.org/)
 
+[awesome-embedded-rust](https://github.com/rust-embedded/awesome-embedded-rust)
+
 * Arduino
   * [avr-rust/ruduino](https://github.com/avr-rust/ruduino) Reusable components for the Arduino Uno.
 * Cross compiling
   * [japaric/rust-cross](https://github.com/japaric/rust-cross) — everything you need to know about cross compiling Rust programs [![build badge](https://api.travis-ci.org/japaric/rust-cross.svg?branch=master)](https://travis-ci.org/japaric/rust-cross)
   * [japaric/xargo](https://github.com/japaric/xargo) — effortless cross compilation of Rust programs to custom bare-metal targets like ARM Cortex-M [![build badge](https://api.travis-ci.org/japaric/xargo.svg?branch=master)](https://travis-ci.org/japaric/xargo)
+* Espressif
+  * [esp-rs](https://github.com/esp-rs) home to a number of community projects enabling the use of the Rust programming language on various SoCs and modules produced by Espressif Systems.
```

Comment:
This will need an exception adding to https://github.com/rust-unofficial/awesome-rust/blob/master/src/main.rs#L43 because there's no stars on an org, but there's enough stars on the repos to make it viable

Reply:
Hello @palfrey! Thanks for your feedback I created a new PR with your suggestions, see #1288.